packages/web/src/components/user-list/UserList.tsx

```diff
@@ -36,6 +38,77 @@ const skeletonData: SkeletonItem[] = range(6).map((index) => ({
   user_id: `skeleton ${index}`
 }))
 
+type UserListItemProps = {
+  userId: ID
+  isLastItem: boolean
+  onClickArtistName: (handle: string) => void
+  onClose: () => void
+  showSupportFor?: ID
+  showSupportFrom?: ID
+}
+
+const UserListItem = ({
+  userId,
+  isLastItem,
+  onClickArtistName,
+  onClose,
+  showSupportFor,
+  showSupportFrom
+}: UserListItemProps) => {
+  const isMobile = useIsMobile()
+  const { data: currentUserId } = useCurrentUserId()
+  const { mutate: followUser } = useFollowUser()
```

Comment:
nice

Reply:
just heads up im a little worried about the big useTracks select in these hooks are now multiplied across every user-list item

Reply:
maybe it makes sense to pass these methods down from the root?

Reply:
ahh I see. Good idea

Also maybe we don't need that as a selector. It can just grab from state in the callback, right?

Reply:
yeah can also chat about this?